src/services/assetTypeManagementServices.js

```diff
@@ -7,12 +7,18 @@ export default {
     getAssetType(assetTypeId) {
         return apiClient.get(`/asset-types/${assetTypeId}`);
     },
+    getFullAssetType(assetTypeId) {
+        return apiClient.get(`/asset-types/${assetTypeId}?full`);
+    },
     createAssetType(data) {
         return apiClient.post(`/asset-types/`, data);
     },
     updateAssetType(assetTypeId, data) {
         return apiClient.put(`/asset-types/${assetTypeId}`, data);
     },
+    updateAssetTypeFields(assetTypeId) {
```

Comment:
You need to pass in data to this function, otherwise how can the backend know what changes to make to the asset type?